### PR TITLE
cfupload: Kinda? Huge Update.

### PR DESCRIPTION
**Pull request by CodeFoxy-Github**
add new apis and types(IDK why i write this 6 hours...😒😒)
- [x] **base64**
- [x] **MIME type**
- [x] **Pastebin**
- [x] **0x0.st**
- [x] **pinata.cloud** IPFS!
- [x] **headers**
- [x] (Deleted cus too bad) ddownload.com

### CODE REVIEW

**Comment by CodeFoxy-Github** [comment]
// if using this ext in penguinmod Wrong base64 will show this
TypeError: uploadFileToLink(...).then is not a function
image

**Comment by CodeFoxy-Github** [comment]
@SharkPool-SP  Review?🤓

**Comment by SharkPool-SP** [comment]
You're not allowed to rename opcodes/remove old stuff due to compatibility issues, if you want them gone, simply just hide the blocks

**Comment by CodeFoxy-Github** [comment]
OK

File: static/extensions/Codefoxy/cfupload.js [hunk 1]
```diff
@@ -1,32 +1,46 @@
-// Name: Upload
-// ID: cfupload
-// Description: Upload files to the Network, can also used with Files extension.
-// By: Codefoxy <https://scratch.mit.edu/users/odavido123Daptoper/>
-// License: MIT AND MPL-2.0
```
Comment (SharkPool-SP) [inline static/extensions/Codefoxy/cfupload.js:5]: Don't remove this

File: static/extensions/Codefoxy/cfupload.js [hunk 2]
```diff
@@ -158,48 +242,102 @@
...
+              fileData = atob(args.data);
+            } catch (e) {
+              return "Base64 error: Invalid data";
+            }
           }
-        }
-      );
-    }
 
-    uploadFileToLink(args) {
```
Comment (SharkPool-SP) [inline static/extensions/Codefoxy/cfupload.js:175]: Idk if a block with this opcode exists, if so, you should not be removing it

**Review by SharkPool-SP — CHANGES_REQUESTED** [review]

**Comment by CodeFoxy-Github** [comment]
# @SharkPool-SP  **Oi!**
#### (lol)

File: static/extensions/Codefoxy/cfupload.js (hunk 2 above)
Comment (CodeFoxy-Github) [reply static/extensions/Codefoxy/cfupload.js:175]: i changed to ``const uploadFileToLinkFunc = (){ }``

**Comment by CodeFoxy-Github** [comment]
yooo

**Pull request merged**